task001/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/kyokomi/emoji"
+
+func main() {
+    emoji.Println("Hello, world :pizza:")
+}
```

Comment:
No newline at the end of file. Try `go fmt .\main.go` it should add one. Run `go fmt` before every commit is a good habit)

The reason is that Unix historically had a convention of all human-readable text files ending in a newline.
Because of this convention, many tools expect the ending newline, including text editors, diffing tools, and other text processing tools.